### PR TITLE
Create Examples for Mock Implementation of Portfolio Drafts API

### PR DESCRIPTION
**Pull request by zachclark-ccpo**
## Overview
mock-atat-internal-api is currently living in the CCPO-ATAT SwaggerHub, and is Private and Unpublished.
These examples provide mock functionality for:

- `GET /portfolioDrafts`
- `POST /portfolioDrafts`
- `GET /portfolioDrafts/{portfolioDraftId}`
- `DELETE  /portfolioDrafts/{portfolioDraftId}`
- `GET /portfolioDrafts/{portfolioDraftId}/portfolio`
- `POST /portfolioDrafts/{portfolioDraftId}/portfolio`
- `GET /portfolioDrafts/{portfolioDraftId}/funding`
- `POST /portfolioDrafts/{portfolioDraftId}/funding`
- `GET /portfolioDrafts/{portfolioDraftId}/application`
- `POST /portfolioDrafts/{portfolioDraftId}/application`
- `POST /portfolioDrafts/{portfolioDraftId}/submit`

There is no mock implementation for the error responses, as SwaggerHub doesn't contain server side validation.

Ticket: **AT-6390**

### CODE REVIEW

File: atat_provisioning_wizard_api.yaml [hunk 1]
```diff
@@ -562,4 +588,72 @@ components:
...
+          - "joe.manager@example.com"
+          - "jane.manager@example.com"
+    FundingStepEx:
+      value:
+        - task_order_number: "12345678910"
+          task_order_file_id: "Mock File ID"
+          csp: 
+            - "aws"
+          clin: 
+            clin_number:  ""
```
[the remark this answers is not in the document]
Comment (zachclark-ccpo) [reply atat_provisioning_wizard_api.yaml:633]: Yes.

File: atat_provisioning_wizard_api.yaml [hunk 2]
```diff
@@ -562,4 +588,72 @@ components:
...
+            pop_start_date: "2021-09-01"
+            pop_end_date: "2022-09-01"
+    ApplicationStepEx:
+      value:
+        - name: "Mock Application"
+          description: "The description of the Mock Application"
+          environments:
+            - name: "environment 1"
+              operators:
+                - first_name: "John"
```
[the remark this answers is not in the document]
Comment (zachclark-ccpo) [reply atat_provisioning_wizard_api.yaml:646]: I will improve the Star Wars references.

[the remark this answers is not in the document]
Comment (jasonburkert-ccpo) [reply atat_provisioning_wizard_api.yaml]: Salacious Crumb

**Comment by jasonburkert-ccpo** [comment]
Well done.
> frozen@carbonite.com

**Review by kylelaker-ccpo — APPROVED** [review]

**Review by kylelaker-ccpo — APPROVED** [review]

**Review by jeffsegal-ccpo — APPROVED** [review]

**Pull request merged**